src/main/java/springboot/webapp/usersmanager/controllers/UserController.java

```diff
@@ -0,0 +1,49 @@
+package springboot.webapp.usersmanager.controllers;
+
+import lombok.AllArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import springboot.webapp.usersmanager.entities.User;
+import springboot.webapp.usersmanager.services.UserService;
+
+import java.util.List;
+
+
+@RestController
+@AllArgsConstructor
+@RequestMapping("/users")
+public class UserController {
+
+    private final UserService userService;
+
+    @GetMapping
+    public ResponseEntity<List<User>> getAll() {
+        return userService.getAll().size() > 0
```

Comment:
in this case, it is normal to return an empty list.
Why do you call the service method 2 times?